src/controllers/exchanger.controller.ts

```diff
@@ -3,9 +3,12 @@ import { ExchangerService } from '../services/exhanger.service';
 import { StatusCode } from '../models/status-codes.model';
 
 export async function getCurrentRate(req: Request, res: Response): Promise<void> {
-  const currentRate = await ExchangerService.getCurrentRate();
-  if (!currentRate) {
-    res.status(StatusCode.BadRequest).json('Invalid status value');
+  try {
+    const currentRate = await ExchangerService.getCurrentRate();
+    res.status(StatusCode.Success).json(currentRate);
+  } catch (error) {
+    const errorMessage = `Error received while getting current rate: ${JSON.stringify(error)}`;
+    console.log(errorMessage);
+    res.status(StatusCode.InternalError).send(errorMessage);
```

Comment:
Seems like console.log left here, in case it is not used as the logger for the project.